### ISSUE TITLE
if php by path. php.ini cannot modification.

### ISSUE THREAD

**Issue by qoli** [bitnami/bitnami-docker-wordpress#66]
i check php.ini in /opt/bitnami/php/lib.
but not.

at last, i "docker exec" input docker, and build a soft link.
`ls -s /opt/.../php/conf/php.ini  /opt/bitnami/php/lib/php`

so that, php.ini is work.

my **docker-compose.yml**
```
version: '2'
services:
  mariadb:
    image: bitnami/mariadb:latest
    volumes:
      - /home/ubuntu/wp/mariadb-persistence:/bitnami/mariadb
  wordpress:
    image: bitnami/wordpress:latest
    depends_on:
      - mariadb
    ports:
      - '80:80'
      - '443:443'
    volumes:
      - /home/ubuntu/wp/wordpress-persistence:/bitnami/wordpress
      - /home/ubuntu/wp/apache-persistence:/bitnami/apache
      - /home/ubuntu/wp/php-persistence:/bitnami/php

```

**Comment by sameersbn** [comment]
The default php configuration files, which includes the `php.ini` file, are generated in the `/bitnami/php` volume when the container is launched for the first time.

If you wish to modify the default php configuration, you can edit the file located at the `/bitnami/php/conf/php.ini` path of the container (or `/home/ubuntu/wp/php-persistence/conf/php.ini` on the host) and then restart the container using `docker-compose restart wordpress`.

Do let us know if this resolves the issue.

**Comment by qoli** [comment]
sure.
before it, i try your say that way.
but, don't work,

so that, i post issue report to you.

`ls -s /opt/bitnami/php/conf/php.ini /opt/bitnami/php/lib/php`

**Comment by sameersbn** [comment]
@qoli I am able to reproduce your issue. We will get back to you with a fix ASAP. Thanks for taking the time to report the issue.

**Comment by sameersbn** [comment]
@qoli the issue has been resolved in the `bitnami/wordpress:4.7.1-r1` release
